### PR TITLE
vmui: fix bug where tenant list was not displayed

### PR DESCRIPTION
**Pull request by Loori-R**
The list of tenants will be displayed if either of the following conditions is met:
- the `useTenantID` flag is set to `true`.
- the `serverUrl` contains tenantID, for example, `/select/<tenantID>/,` and there is more than one tenant in the response.

#4160

### CODE REVIEW

File: app/vmui/packages/vmui/src/utils/tenants.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const regexp = /(\/select\/)(\d+|\d.+)(\/)(.+)/;
+
+export const replaceTenantId = (serverUrl: string, tenantId: string) => {
+  return serverUrl.replace(regexp, `$1${tenantId}/$4`);
+};
+
+export const getTenantIdFromUrl = (url: string) => {
+  return (url.match(regexp) || [])[2];
+};
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/utils/tenants.ts:9]: I think it is not a good idea to return undefined. You can check the length of the array and return string. This string can be empty or can have a value

Comment (Loori-R) [reply app/vmui/packages/vmui/src/utils/tenants.ts]: Sure, good catch. I've made the correction

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Comment by valyala** [comment]
FYI, this pull request has been included in [v1.91.0](https://docs.victoriametrics.com/CHANGELOG.html)

**Pull request merged**